components/raft_log_engine/src/engine.rs

```diff
@@ -222,6 +224,11 @@ impl FileSystem for ManagedFileSystem {
             })
         }
     }
+
+    fn delete<P: AsRef<Path>>(&self, _path: P) -> IoResult<()> {
+        // TODO(tabokie)
+        Ok(())
```

Comment:
Do we need to mark it `todo!()` to avoid side effects by now?

Reply:
I propogate it to the inner `delete`, and should work now. cc @tabokie

Reply:
After all, this will be replaced with the content in https://github.com/tikv/tikv/pull/12892.. So, just let it work...